.atomist/editors/RugEditorsPredicates.ts

```diff
@@ -19,19 +19,31 @@ import { Project } from "@atomist/rug/model/Project";
 import deprecated from "deprecated-decorator";
 
 export function isRugArchive(p: Project): boolean {
-    return p.fileExists(".atomist/manifest.yml");
-}
-
-export function isSetUpForTypeScript(p: Project): boolean {
+    if (p.fileExists(".atomist/manifest.yml")) {
+        p.editWith("ConvertManifestToPackageJson", {});
```

Comment:
I thought about that to but figured, "@jessitron would want this to do the smart thing automatically"!